### PR TITLE
feat: check runtime imports in `make pre-commit`

### LINKED ISSUE #3137: Need to check runtime imports in `make pre-commit`
### Problem

We don't verify runtime imports in `make pre-commit`. They are checked on CI only.

### Possible Solution

Need to add the following commands to `Makefile`:

- `./target/release/wasm-proc --check-runtime-imports target/release/wbuild/vara-runtime/vara_runtime.compact.wasm`
- `./target/release/wasm-proc --check-runtime-imports target/release/wbuild/gear-runtime/gear_runtime.compact.wasm`

### PR DESCRIPTION
**Pull request by shamilsan**
Resolves #3137

@gear-tech/dev

### CODE REVIEW

**Review by NikVolf — APPROVED** [review]

**Review by clearloop — APPROVED** [review]
https://github.com/gear-tech/gear/blob/master/scripts/src/check.sh#L17

plz provide help message as well

File: scripts/src/check.sh [hunk 1]
```diff
@@ -25,3 +25,19 @@ gear_check() {
   echo "  >> Check crates that use runtime with 'fuzz' feature"
   cargo check "$@" -p runtime-fuzzer -p runtime-fuzzer-fuzz
 }
+
+runtime_imports() {
+    if [ ! -f target/debug/wasm-proc ]; then
+        cargo build -p wasm-proc
+    fi
+
+    if [ ! -f target/debug/wbuild/gear-runtime/gear_runtime.compact.wasm ]; then
```
Comment (ukint-vs) [inline scripts/src/check.sh:34]:
```suggestion
    if [ ! -f target/release/wbuild/gear-runtime/gear_runtime.compact.wasm ]; then
```

File: scripts/src/check.sh [hunk 2]
```diff
@@ -25,3 +25,19 @@ gear_check() {
...
+        cargo build -p gear-runtime
+    fi
+    ./target/debug/wasm-proc --check-runtime-imports target/debug/wbuild/gear-runtime/gear_runtime.compact.wasm
+
+    if [ ! -f target/debug/wbuild/vara-runtime/vara_runtime.compact.wasm ]; then
```
Comment (ukint-vs) [inline scripts/src/check.sh:39]:
```suggestion
    if [ ! -f target/release/wbuild/vara-runtime/vara_runtime.compact.wasm ]; then
```

File: scripts/src/check.sh [hunk 3]
```diff
@@ -25,3 +25,19 @@ gear_check() {
...
+        cargo build -p vara-runtime
```
Comment (ukint-vs) [inline scripts/src/check.sh:40]: why you build runtimes in debug?

File: scripts/src/check.sh (hunk 1 above)
Comment (shamilsan) [reply scripts/src/check.sh:34]: Discussed

File: scripts/src/check.sh (hunk 2 above)
Comment (shamilsan) [reply scripts/src/check.sh:39]: Discussed

**Review by breathx — APPROVED** [review]

**Pull request merged**